flutter/tonic/dart_library_loader.h

```diff
@@ -31,11 +31,14 @@ class DartLibraryLoader {
   ~DartLibraryLoader();
 
   // TODO(dart): This can be called both on the main thread from application
-  // solates or from the handle watcher isolate thread.
+  // isolates or from the handle watcher isolate thread.
   static Dart_Handle HandleLibraryTag(Dart_LibraryTag tag,
                                       Dart_Handle library,
                                       Dart_Handle url);
 
+  static bool IsSourceUrlModified(const char* source_url,
+                                  int64_t since);
```

Comment:
Is there any reason to put this function here?  It seems like a general function that doesn't have anything to do with Dart or Tonic.